.NET/Microsoft.Recognizers.Definitions/German/DateTimeDefinitions.cs

```diff
@@ -703,7 +707,7 @@ public static class DateTimeDefinitions
 			{ @"zweitausender", 2000 }
 		};
 		public const string DefaultLanguageFallback = @"DMY";
-		public static readonly string[] DurationDateRestrictions = { "heute", "jetzt", "momentan", "aktuell", "gerade" };
+		public static readonly string[] DurationDateRestrictions = { "today", "now" };
```

Comment:
Why back in English?

Reply:
I translated them before, but they were not used. So I put back the English terms to avoid confusion.